lib/translation_maps/locations.properties

```diff
@@ -0,0 +1,709 @@
+ACAD-BLDG = Submit request for annexed material
```

Comment:
We can't get this from a web service call, right?

Reply:
I am not sure honestly, can check with Maryam

Reply:
ok, if she says yes we can follow up with a new PR to replace 👍

Reply:
exactly what I was thinking 👍 :)